launchable/test_runners/nunit.py

```diff
@@ -119,13 +123,13 @@ def subset(client, report_xmls):
     Parse an XML file produced from NUnit --explore option to list up all the viable test cases
     """
 
-    def on_element(e: Element):
-        build_path(e)
-        if e.name == "test-case":
-            client.test_path(_replace_fixture_to_suite(e.tags['path']))
+    def on_element(e: ET.Element, parent: ET.Element):
```

Comment:
Why does this method need `parent? Doesn't `e` have a `parent` attribute?

Reply:
In elementTree API, each element does not have parent attribute.

https://docs.python.org/ja/3/library/xml.etree.elementtree.html#xml.etree.ElementTree.Element